corehq/apps/saved_reports/models.py

```diff
@@ -737,14 +737,15 @@ def _get_and_send_report(self, language, emails):
                             mock_query_string_parts.append(urlencode(report_config.filters, True))
                             mock_query_string_parts.append(urlencode(report_config.get_date_range(), True))
                         mock_request.GET = QueryDict('&'.join(mock_query_string_parts))
-                        date_range = report_config.get_date_range()
-                        start_date = datetime.strptime(date_range['startdate'], '%Y-%m-%d')
-                        end_date = datetime.strptime(date_range['enddate'], '%Y-%m-%d')
-
-                        datespan = DateSpan(start_date, end_date)
                         request_data = vars(mock_request)
                         request_data['couch_user'] = mock_request.couch_user.userID
-                        request_data['datespan'] = datespan
+                        if report_config.report_slug != ApplicationStatusReport.slug:
```

Comment:
It'd be more flexible to have this check whether the behavior (date range) is present, rather than check the slug. Would there be a downside to checking `if 'start_date' in date_range` instead?

Reply:
I was thinking of that, but I didn't know if there was a case where that would be empty but the report might still require it for some reason. I didn't want to break any other reports by doing this, so I specifically added check for this report.